tests/auto/make_evmtests.py

```diff
@@ -21,7 +26,7 @@ def pretty(value, htchar=' ', lfchar='\n', indent=0, width=100):
             for item in value
         ]
         return '(%s)' % (','.join(items) + lfchar + htchar * indent)
-    elif type(value) in (str, unicode):
+    elif type(value) in (str, str):
```

Comment:
It should rather be `elif isinstance(value, str):`

Reply:
👍 ;-)